.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 
 default_language_version:
-    python: python3.7
+    python: python3
```

Comment:
we want some control on the version here for reproducibility across developer machines. Forcing to 3.7 was good I think

But now, I see it want changes on geoh5py too from 3.7 to 3 ???

Reply:
It was breaking with python3.7

Reply:
the last commit on develop was about forcing to 3.7
Please do not change.
We do not want pre-commit to auto-fix for a syntax which is reserved to 3.9 and not compatible with 3.7